src/cogent3/maths/stats/special.py

```diff
@@ -295,6 +297,8 @@ def erfc(a):
         return y
 
 
+
```

Comment:
you should run black and isort against the code. In the root directory for the project
```
$ black .; isort .
```
these ensure formatting, white space etc.. all match the project standards